nginx-controller/nginx/config.go

```diff
@@ -49,6 +49,7 @@ type Config struct {
 	HealthCheckMandatory          bool
 	HealthCheckMandatoryQueue     int64
 	SlowStart                     string
+	LogLevel                      string
```

Comment:
Please rename to MainLogLevel or MainErrorLogLevel -- settings that go in the main nginx config file have a "Main" prefix.